webauthn4j-core/src/main/java/com/webauthn4j/validator/attestation/statement/androidsafetynet/AndroidSafetyNetAttestationStatementValidator.java

```diff
@@ -162,7 +162,7 @@ private void validateNonce(@Nullable String nonce, @NonNull byte[] authenticator
         ByteBuffer buffer = ByteBuffer.allocate(authenticatorData.length + clientDataHash.length);
         byte[] data = buffer.put(authenticatorData).put(clientDataHash).array();
         byte[] hash = MessageDigestUtil.createSHA256().digest(data);
-        if (!Arrays.equals(hash, Base64Util.decode(nonce))) {
+        if (!MessageDigest.isEqual(hash, Base64Util.decode(nonce))) {
```

Comment:
In my understanding, timing attack is a way to find out a secret value. 
In this case, as the source of `hash` is known data to client side(potential attacker) because it is part of a message, there is no risk of timing attack and it is OK to use `Arrays.equals` instead of `MessageDigest.isEqual` here.
This is a false positive warning of snyk.